style.css

```diff
@@ -1,1333 +1,1206 @@
-    position: static;
-    width: 240px;
-    height: 90px;
-    font-family: "Crete Round", sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 45px;
-    line-height: 55px;
-    color: #ff6b00;
-    margin: 12px 44px;
-  }
-
-  .support {
-    position: relative;
-    width: 250px;
-    height: 17px;
-    left: 1px;
-    top: 3px;
-    font-family: "poppins", sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 19px;
-    line-height: 12px;
-    display: flex;
-    align-items: center;
-    color: #fff5e1;
-    flex: none;
-    order: 1;
-    margin: 20px 44px;
+.work {
+  font-size: 40px;
+  color: #1c1a19;
+  text-align: center;
+  margin-top: 120px;
+}
+
+.text {
+  color: #091e42;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.head {
+  margin-top: 24px;
+  margin-bottom: 12px;
+}
+
+hr {
+  width: 327px;
+  color: #dbd8d7;
+  margin: 24px 0 80px 28px;
+  text-align: center;
+}
+
+.temp2 {
+  background-image: url(./images/MaskGroup.png);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-repeat: no-repeat;
+}
+
+.size {
+  height: 386px;
+  width: 327px;
+  align-self: center;
+}
+
+.temp {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.data {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.tech {
+  display: flex;
+  padding-top: 22px;
+  gap: 15px;
+}
+
+.tech li {
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+}
+
+.btn {
+  padding: 12px;
+  color: #fff;
+  background-color: #ff6b00;
+  border: none;
+  font-size: 17px;
+  width: 124px;
+}
+
+.li-btn {
+  padding-bottom: 12px;
+}
+
+.btn-2 {
+  width: 100%;
+}
+
+.btn-1st {
+  margin: 0;
+}
+
+.text-2 {
+  color: #fff;
+  height: 89px;
+  width: 288px;
+  margin-bottom: 12px;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  gap: 50px;
+}
+
+.pad {
+  padding-left: 15px;
+  background: linear-gradient(179.4deg, rgba(38, 38, 38, 0) 0.81%, rgba(38, 38, 38, 0.9) 84.03%);
+}
+
+.card-heading {
+  margin-top: 84px;
+  margin-bottom: 12px;
+  color: #fff;
+  text-align: left;
+}
+
+.tech-2 {
+  padding-top: 0;
+  color: #fff;
+}
+
+.tech-2 li {
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.about {
+  color: #ff6b00;
+  padding-top: 120px;
+  font-size: 56px;
+  line-height: 64px;
+  font-family: "Crete Round", sans-serif;
+}
+
+.container-2 {
+  display: flex;
+  background-color: #1c1a19;
+  margin-top: 120px;
+  height: 600px;
+  padding-left: 24px;
+  background-image: url(./image/Frame.png);
+  background-repeat: no-repeat;
+  background-position: 254px 300px;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.btn-res {
+  width: 161px;
+}
+
+.text-res {
+  color: #fff;
+}
+
+.about-section {
+  background-color: #1c1a19;
+  color: #fff;
+}
+
+.puOut li {
+  padding: 10px 10px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-right: 10px;
+  border: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] [Optional] Kindly add a margin to your list buttons in the mobile view
    
![image](https://user-images.githubusercontent.com/66097371/163544955-fa54a27b-e519-4bd4-a514-057ed14fa1d6.png)